lib/tasks/one_offs.rake

```diff
@@ -4,14 +4,17 @@ namespace :one_offs do
   desc "backfill teacher catchement iso country code"
   task backfill_teacher_catchment_iso_country_code: :environment do
     uk_country = ISO3166::Country.find_country_by_any_name("United Kingdom")
+
+    NPQApplication.where.not(teacher_catchment: [nil, "another"]).find_each do |npq_application|
+      npq_application.update!(
+        teacher_catchment_iso_country_code: uk_country.alpha3,
+        teacher_catchment_country: uk_country.iso_short_name,
+      )
+    end
+
     NPQApplication.where.not(teacher_catchment_country: nil).find_each do |npq_application|
```

Comment:
```suggestion
    NPQApplication.where.not(teacher_catchment_country: nil).where(teacher_catchment: [nil, "another"]).find_each do |npq_application|
```
sorry missed this before, to avoid going through the above again as now they're set